Resources/Locale/en-US/fluids/components/bucket-component.ftl

```diff
@@ -1,2 +1,3 @@
 bucket-component-bucket-is-empty-message = Bucket is empty
-bucket-component-mop-is-full-message = Bucket is full
```

Comment:
"bucket is full" string did not match purpose of the message - should have said "mop is full."